### ISSUE TITLE
Partial matching

### ISSUE THREAD

**Issue by esso23** [hez2010/BlazorRouter#13]
Hi,
would it be possible to implement partial matching on Routes? For example React Router uses partial matching by default and you can use Exact parameter to specify that you want an exact match.

**Context:**
I have a nested route looking something like this:

In MainLayout:
```html
// ...
<Route Template="/station/{StationId:int}"> // I want partial matching here
    <StationComponent />
</Route>
// ...
```

StationComponent:
```html
// ...
<Route Template="/station/{StationId:int}">
    <Component1 />
</Route>
<Route Template="/station/{StationId:int}/general">
    <Component1 />
</Route>
<Route Template="/station/{StationId:int}/somethingElse">
    <Component2 />
</Route>
<Route Template="/station/{StationId:int}/somethingElse2">
    <Component3 />
</Route>
<Route Template="/station/{StationId:int}/somethingElse3Etc">
    <Component4 />
</Route>
// ...
```

I don't want to add the route to MainLayout everytime I add a new Route to StationComponent.
Great library btw. These nested routes were giving me a headache when using Blazor built-in router until I found this lib.

**Comment by hez2010** [comment]
Sorry for the delay.
I'll implement it for the next version.

**Comment by hez2010** [comment]
It already support it with below patterns:
```html
<Switch>
  <Route Template="/station/{StationId:int}">
    <Component1 />
  </Route>

  <Route Template="/station/{StationId:int}/*">
    <Switch>
      <Route Template="/station/{StationId:int}/general">
        <Component1 />
      </Route>
      <Route Template="/station/{StationId:int}/somethingElse">
        <Component2 />
      </Route>
      <Route Template="/station/{StationId:int}/somethingElse2">
        <Component3 />
      </Route>
      <Route Template="/station/{StationId:int}/somethingElse3Etc">
        <Component4 />
      </Route>
    </Switch>
  </Route>
</Switch>
```

What you need to do is to wrap your `StationComponent` with a `Switch`.

I'll update the docs.

**Comment by hez2010** [comment]
Fixed in 12d33590a482fc3c92ae8a6581459cb4955e5f64, please refer to the new document.
